test/fixtures/typescript/test-experimental-decorators.ts

```diff
@@ -0,0 +1,14 @@
+function sealed(constructor: Function) {
+    Object.seal(constructor);
+    Object.seal(constructor.prototype);
+}
+
+@sealed
+class BugReport {
+    type = "report";
+    title: string;
+
+    constructor(t: string) {
+        this.title = t;
+    }
+}
```

Comment:
This and the other TypeScript fixtures should presumably be indented with two spaces, like our JavaScript files. We should add TypeScript files to our eslint config.

Reply:
I dont think eslint supports ts out of the box without a plugin

Reply:
Well at the very least let’s fix the indentation for all the TypeScript files to 2 spaces.